app/models/spree/package_builder.rb

```diff
@@ -84,11 +84,24 @@ def convert_package_to_item_packages_array(solidus_package)
     end
 
     # Generates an array of Package objects based on the quantities and weights of the variants in the line items
-    def packages(solidus_package)
+    def to_packages(solidus_package)
       active_shipping_packages = []
-      weights = convert_package_to_weights_array(solidus_package)
+
+      ci_without_boxes = []
```

Comment:
I would prefer if we didn't introduce a new name for `product_packages` here as `boxes`. We also generally avoid abbreviating things in names. It isn't immediately clear what `ci` is when reading this code.